cdbot/cogs/general.py

```diff
@@ -34,6 +37,17 @@ async def on_ready(self):
             f"\nAuthor: **{latest.author}** on {date}"
         )
 
+    @Cog.listener()
+    async def on_member_join(self, member):
+        join_msg_channel = self.bot.get_channel(WELCOME_CHANNEL)
```

Comment:
Single use variable

Reply:
This isn't a problem, it's to prevent unnecessarily long lines. Readability counts.